packages/neuron-ui/src/locales/en.json

```diff
@@ -689,7 +689,8 @@
         "compensated-period": "Compensation Period",
         "days-hours": "{{days}} days {{hours}} hours",
         "no-deposit": "No Deposit",
-        "no-completed": "No Completed"
+        "no-completed": "No Completed",
+        "no-ckb": "No other CKB to pay for transaction fee that unlock DAO"
```

Comment:
`No other CKB` is hard to understand, how about `Balance is not enough for transaction fee to unlock the deposit`